apps/api/src/app/vercel-integration/vercel-integration.controller.ts

```diff
@@ -0,0 +1,32 @@
+import { Controller, Post, Param, UseInterceptors, UseGuards, ClassSerializerInterceptor, Body } from '@nestjs/common';
+import { ApiTags } from '@nestjs/swagger';
+import { IJwtPayload } from '@novu/shared';
+import { JwtAuthGuard } from '../auth/framework/auth.guard';
+import { UserSession } from '../shared/framework/user.decorator';
+import { SetupIntegrationRequestDto } from './dtos/setup-integration-request.dto';
+import { SetupIntegrationResponseDto } from './dtos/setup-integration-response.dto';
+import { SetupIntegrationCommand } from './usecases/setup-integration/setup-integration.command';
+import { SetupIntegration } from './usecases/setup-integration/setup-integration.usecase';
+
+@Controller('/vercel-integration')
+@UseInterceptors(ClassSerializerInterceptor)
+@UseGuards(JwtAuthGuard)
+@ApiTags('Integrations')
```

Comment:
We can exclude this from the swagger documentation. (Excluding the entire controller)